db/data.js

```diff
@@ -13,6 +13,7 @@ export const initialProjects = [
       { id: 4, desc: "Sand the shelf to smooth out rough edges." },
       { id: 5, desc: "Paint or stain the shelf as desired." },
     ],
+    image: "http://localhost:3000/images/dummy-image-diy.jpg",
```

Comment:
```suggestion
    image: "../public/images/dummy-image-diy.jpg",
```

Reply:
correct this in all data objects :) ↓

Reply:
thanks. it actually missed another "/" to work, but I figured this out myself 😀 💪 
image: "/../public/images/dummy-image-diy.jpg",